Core/include/Acts/Seeding/SeedfinderConfig.hpp

```diff
@@ -1,6 +1,6 @@
 // This file is part of the Acts project.
 //
-// Copyright (C) 2018 CERN for the benefit of the Acts project
+// Copyright (C) 2020 CERN for the benefit of the Acts project
```

Comment:
What's the convention here? Should I have used `2018-2020` instead?

Reply:
Hi, yeah - usually we tried to do 2018-2020, but we are not super strict, we tried a script once to do this automatically

Reply:
Done. :+1: